src/Catalyst.jl

```diff
@@ -23,7 +23,7 @@ using RuntimeGeneratedFunctions
 RuntimeGeneratedFunctions.init(@__MODULE__)
 
 import Symbolics: BasicSymbolic
-using Symbolics: iscall
```

Comment:
I figured we used `sorted_arguments` enough that it might make sense to import the function (happy to revert, not sure if we have any guidelines when we want to import/not import a function)